src/components/App/App.module.css

```diff
@@ -1,9 +1,13 @@
 .wrap {
     display: block;
     margin: auto;
-    width: 300px;
+    width: 500px;
+    background-color: #ffffff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5693637)